Sources/TokamakCore/Views/View+DataChanges.swift

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Szymon on 6/8/2023.
+//
+
+import Foundation
+import OpenCombineShim
+
+extension View {
```

Comment:
Can these be split into separate `OnReceiveModifier` and `OnChangeModifier` files?

Reply:
done